### PR TITLE
Feature/29705

### PR DESCRIPTION
**Pull request by kjugi**
+ Update `.gitignore` list
+ Add necessary dependencies
+ Update global styles and move to other dir

https://dev.snowdog.pro/issues/29705

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -160,23 +166,31 @@ gulp.task('fractal:build', ['sass'], () => {
...
-            .pipe(sass({outputStyle: 'compressed'}))
-            .pipe(postcss(processors))
-            .pipe(sourcemaps.write(paths.maps))
-            .pipe(gulp.dest(paths.css))
-    })
+    gulp.watch([
+        paths.components + '/**/*.scss',
+        'docs/**/*.scss'
+    ],
+    () => {
```
Comment (Igloczek) [inline gulpfile.js]: Combine 172 and 173 together.

File: docs/styles.scss [hunk 2]
```diff
@@ -48,6 +46,7 @@
 @import '../components/blocks/video-box/video-box';
 
 // documentation style
+@import 'docs-styles/base';
```
Comment (Igloczek) [inline docs/styles.scss]: Same changes as in #2

Comment (kjugi) [inline docs/styles.scss]: Renamed

File: gulpfile.js (hunk 1 above)
Comment (kjugi) [inline gulpfile.js]: Think I combined it together

File: docs/styles.scss (hunk 2 above)
Comment (kjugi) [inline docs/styles.scss]: renamed

**Pull request merged**